### ISSUE TITLE
Is pretrained model availble?

### ISSUE THREAD

**Issue by shouldnotfail** [ika-rwth-aachen/EviLOG#9]
Hi! 
Thanks for sharing the good work. I am interested in Deep ISM for LiDAR for a long time, and want to evaluate the neural network on the dataset which is recorded by myself and has no labels for OGM, and see the generalization performance.  Vielen Dank im Voraus!

**Comment by shouldnotfail** [comment]
Ok! Then I will apply the access to data and train it from scratch. Thanks again!
